mycroft/configuration/mycroft.conf

```diff
@@ -119,7 +119,7 @@
     // this config value is not present in mycroft-core ()internet is required)
     // mycroft-lib expects that some instances will be running fully offline
     "wait_for_internet": true,
-
+    "run_gui_file_server": false,
```

Comment:
i feel this doesnt really belong in skills section, can it be moved to gui ?

Reply:
I put it here since the skills module is the one running the file server, so that's the module where this config is referenced. I can put it in gui though if that makes more sense for reading config